### ISSUE TITLE
Chatting didn't take Nancy off Vacation

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#132]
Nancy got sent on vacation, then mentioned it in chat, but that chat didn't take her off vacation.

My recollection was that any command takes one off vacation, although it could be that any _action_ takes one off vacation.

With the UI the way it is and with people not familiar with vacation, the rule should probably be that anything takes one off...

**Comment by ctm** [comment]
```
GrizzB blinds 2000
NNR13 blinds 4000
2 B > JADC    29775    0  [  ] [  ]
3     GrizzB  44925 2000  [  ] [  ]
6     NNR13    8500 4000  [  ] [  ]
JADC calls
We're now on Level 14
GrizzB raises 4000 to 8000
...
NNR13 has gone on vacation
...
Dealing 6000 12000 Deuce-to-Seven Triple Draw

JADC blinds 3000
GrizzB blinds 6000
2     JADC     2325 3000  [  ] [  ] [  ] [  ] [  ]
3     GrizzB  71150 6000  [  ] [  ] [  ] [  ] [  ]
6 B >V NNR13   5500    0  [  ] [  ] [  ] [  ] [  ]
NNR13 folds
NNR13: ok, it's mad at me, it folds me automatically
JADC calls and is all-in
```
The comment should have taken Nancy off vacation, but it didn't.

**Comment by ctm** [comment]
Aha.  Chat short-circuits the processing of commands. I think that was a change I made long ago, probably around when I removed the IRC interface.

If I make it so chat takes a player off vacation, that should probably be a temporary change, because a better UI would make it superfluous at best.

OTOH, now that UIKit is installed, it probably makes more sense to just put up a dialog that makes it super obvious...
